### PR TITLE
Release 0.9.1

### PR DESCRIPTION
**Pull request by xerial**
Various fixes for passing `mvn deploy` command
- Fixed errors at `mvn javadoc:javadoc` phase

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ For the information of the older versions, see <https://github.com/treasure-data
 ### For Maven Users
 
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.treasuredata.client/td-client/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.treasuredata.client/td-client/)
-[![Javadoc](https://javadoc-emblem.rhcloud.com/doc/com.treasuredata.client/td-client/badge.svg)](http://www.javadoc.io/doc/com.treasuredata.client/td-client)
+[![javadoc](https://javadoc.io/badge2/com.treasuredata.client/td-client/javadoc.svg)](https://javadoc.io/doc/com.treasuredata.client/td-client)
```
Comment (xerial) [inline README.md:27]: javadoc badge link was broken

File: pom.xml [hunk 2]
```diff
@@ -180,6 +180,12 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
```
Comment (xerial) [inline pom.xml:185]: Since JDK9, javax annotaiton (e.g., `@Generated`) needs to be explicitly imported

File: pom.xml [hunk 3]
```diff
@@ -410,12 +416,13 @@
...
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
-        <version>2.10.3</version>
+        <version>3.2.0</version>
         <configuration>
           <source>${project.build.targetJdk}</source>
           <encoding>${project.build.sourceEncoding}</encoding>
           <maxmemory>${project.build.jvmsize}</maxmemory>
           <additionalparam>${javadoc.opts}</additionalparam>
+          <sourcepath>${project.basedir}/src/main;${project.basedir}/target/generated-sources/annotations</sourcepath>
```
Comment (xerial) [inline pom.xml:425]: Need to include classes generated by Immutables

Comment (shimamoto) [reply pom.xml:425]: Ah, Java 8 gives a warning, but Java 11 gives a fatal error

**Pull request merged**